File: Code/WorkspaceB/Python Scripts/MONTRANS.py
'''input
2
9 77 69
98 99 0
'''
tc=eval(input().strip())
for tl in range (0,tc):
	s1=input().strip().split(' ')
	a=int(s1[0])
	b=int(s1[1])
	c=int(s1[2])
	ans=0
	f=float((str(a)+'.'+str(b)))
	m=f
	cnt=0
	cd=float('0.'+str(c))
	while cnt<=10000 and (a>0 or b>=c):
		if(b>=c):
			b-=c
		elif b<c:
			a-=1
			b+=100
			b-=c
		
		a,b=b,a
		cnt+=1
		f=float((str(a)+'.'+str(b)))
		if f>m:
			m=f
			ans=cnt

	print (ans)

File: Code/WorkspaceB/C++_Files/CC/MONTRANS.cpp
/*input
2
9 77 69
98 99 69
*/
#include <iostream>
#include <cstdio>
using namespace std;
#define endl '\n'

//Created by Shreyans Sheth [bholagabbar] using Sublime 3 and SublimeInput Plugin

int main()
{
	int t;
	scanf("%d",&t);
	while(t--)
	{
		int a,b,c;
		scanf("%d%d%d",&a,&b,&c);
		int ans=0,cnt=0;
		float f=(float)(a*100.0+b);
		float max=f;
		while(cnt<10000 && (a>0||b>=c))
		{
			if(b>=c)
			{
				b-=c;
			}
			else
			{
				b+=100;
				b-=c;
				a-=1;
			}
			swap(a,b);
			cnt++;
			f=(float)(a*100.0+b);
			if(f>max)
			{
				max=f;
				ans=cnt;
			}
		}
		printf("%d\n",ans);
	}	

	return 0;
}